test/integration/capacity.test.ts

```diff
@@ -90,8 +90,17 @@ describe('capacity', () => {
       expect(capacity.latestTokenId).to.equal(mockTokenId)
       expect(capacity.originalTokenId).to.equal(mockTokenId)
     })
+
+    it('it should get transactions', async () => {
+      identitySelfMock()
+      const response = await get(app, `/capacity/${seededCapacityId}/creation/${seededTransactionId}`)
```

Comment:
good idea to assert the contents of `response.body` too to make sure it's returning what we want

Reply:
yea, I would say even good to assert all properties so then we make a change test will flag up.